### DISCUSSION TITLE
build Schannel questions

### DISCUSSION THREAD

**Discussion by imba-tjd** [curl/curl-for-win#76] [Q&A] [answered]
1. Is there a Schannel build binary available for download? I searched but didn't find. I know MS Win contains one, but it can't be downloaded on the website
2. I tried GitHub Actions scripts in this repo. `CW_CONFIG=main-win-gcc-mini`. It turns out `Protocol "https" not supported`. Does it mean I must use windows to build Schannel?
3. I tried `CW_CONFIG=main-win-gcc-pico`. It turns out supporting `ipfs ipns`, which is different from the example in Readme.

**Comment by vszakats** [comment, marked as answer]
> 1. Is there a Schannel build binary available for download? I searched but didn't find. I know MS Win contains one, but it can't be downloaded on the website

The build shipping with Windows by Microsoft are not provided by this project.

> 2. I tried GitHub Actions scripts in this repo. `CW_CONFIG=main-win-gcc-mini`. It turns out `Protocol "https" not supported`. Does it mean I must use windows to build Schannel?

Thanks for reporting this.

No, building with Schannel should work from any host OS.

This was a bug caused by cfb67eea57060aabaf406c8f211ce2d2fbb07341, that accidentally disabled
Schannel in more combinations than it was supposed to.

Pushed the fix e3c75109bf28c67c4e0a951f10e7c684abb8b88c.

Does it still happen with the latest `main` branch?

> 3. I tried `CW_CONFIG=main-win-gcc-pico`. It turns out supporting `ipfs ipns`, which is different from the example in Readme.

The option to disable IPFS is available only in the not yet released
curl 8.11.0 version. You can build it with `-dev` instead of `-main`,
until its released.

The README reflects the upcoming version.

**Reply by imba-tjd** [reply]
Thank you. I tried again and it works.
